nbs/01_augment.ipynb

```diff
@@ -40,7 +40,7 @@
     "from fastai2.torch_basics import *\n",
```

Comment:
<p>Maybe the AudioBlock definition should be moved to core nb. This feels out of place here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rbracco/fastai2_audio/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='217'/>